src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskReplicationMIControllerTest.java

```diff
@@ -113,7 +129,95 @@ public void user_should_configure_task_and_configure_action_recorded_in_replica_
     }
 
     @Test
-    public void user_should_claim_task_and_claim_action_recorded_in_replicate_db() {
+    public void user_should_claim_task_and_claim_action_recorded_in_reportable_task() {
+
+        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("processApplication",
+                                                                       "Process Application");
+        initiateTask(taskVariables);
+
+        common.setupWAOrganisationalRoleAssignment(caseworkerCredentials.getHeaders(), "tribunal-caseworker");
+
+        String taskId = taskVariables.getTaskId();
+
+        Response resultTaskReportable = restApiActions.get(
+            ENDPOINT_BEING_TESTED_REPORTABLE,
+            taskId,
+            caseworkerCredentials.getHeaders()
+        );
+
+        resultTaskReportable.prettyPrint();
+        resultTaskReportable.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .body("reportable_task_list.size()", equalTo(1))
+            .body("reportable_task_list.get(0).task_id", equalTo(taskId))
+            .body("reportable_task_list.get(0).state", equalTo("UNASSIGNED"))
+            .body("reportable_task_list.get(0).updated_by", notNullValue())
+            .body("reportable_task_list.get(0).updated", notNullValue())
+            .body("reportable_task_list.get(0).update_action", equalTo("Configure"))
+            .body("reportable_task_list.get(0).first_assigned_date", nullValue())
+            .body("reportable_task_list.get(0).first_assigned_date_time", nullValue())
+            .body("reportable_task_list.get(0).wait_time_days", nullValue())
+            .body("reportable_task_list.get(0).wait_time", nullValue())
```

Comment:
Could we check the dates here? Not exact amounts but as long it's an actual date within 5 mins.

Reply:
same for updated_by etc

Reply:
Did date checks now

Reply:
Looks good